ElectronClient/app/gui/NoteList.jsx

```diff
@@ -117,6 +118,15 @@ class NoteListComponent extends React.Component {
 			}
 		};
 
+		const onTitleDoubleClick = async (event, item) => {
+			if (this.props.watchedNoteFiles.indexOf(item.id) < 0) {
+				this.props.dispatch({
+					type: 'WINDOW_COMMAND',
+					name: 'commandStartExternalEditing',
+				});
+			}
+		};
+
```

Comment:
I'm not sure, if this is a good idea. We might want to use a double-click for a different action in the future. But this is Laurent's decision.

Reply:
Hi Tessus, thanks for your comments.

Apologies, I didn't bring up the idea in the forum before,  I read in the contribution instructions that small changes could be proposed on risk.

The double-click action mimics the behaviour of another well known note-taking tool I have used. It seemed natural to me, but perhaps that's just because I am familiar with it.

Actually, many times in Joplin I double clicked on notes (to open, and of course nothing happened), that's what prompted me to try to add it!

In any case, that part can be easily reverted if necessary

Reply:
No worries, you are right. Smaller PRs can be submitted without discussing them on the forum first. It's just less or no risk when doing so.

The double-click feedback was just a personal opinion, so we have to wait for @laurent22's input anyway. Maybe there's a bettter action for double-click, but maybe not.

It would be great to have some sort of action preferences to define such action shortcuts, but this might be more complex thus an idea for the future.

Reply:
Good idea